src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkAndDeleteModal.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import i18n from '@dhis2/d2-i18n';
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+} from '@dhis2/ui';
+import log from 'loglevel';
+import { useDataEngine, useAlert } from '@dhis2/app-runtime';
+import { useMutation, useQueryClient } from 'react-query';
+import { ReactQueryAppNamespace } from 'capture-core/utils/reactQueryHelpers';
+import type { Props } from './UnlinkAndDeleteModal.types';
+
+export const UnlinkAndDeleteModal = ({
+    setOpenModal,
+    eventId,
+    originEventId,
+}: Props) => {
+    const dataEngine = useDataEngine();
+    const queryClient = useQueryClient();
+    const { show: showErrorAlert } = useAlert(
+        i18n.t('An error occurred while unlinking and deleting the event.'),
+        { critical: true },
+    );
+
+    const deleteEvent = async () => {
+        const mutation = {
+            resource: 'tracker?async=false&importStrategy=DELETE',
```

Comment:
I'm wondering if a single DELETE request is sufficient, or if we need an additional one to delete the relationship. Do you know how the API handles this? Specifically, does deleting the event also remove the associated relationship from the database? 🤔

Reply:
@simonadomnisoru I checked whether I still got data for the relationship ship from the API if I deleted the event. The relationshipship data seemed to me to be deleted, but it might be worth double-checking.